plugins/node/instrumentation-kafkajs/src/propagator.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
```

Comment:
We need to add a Copyright line to keep Aspecto's original license header here and in all other files to comply with the license.

Reply:
Added Aspecto to the copyright line (and modified the eslint pattern)